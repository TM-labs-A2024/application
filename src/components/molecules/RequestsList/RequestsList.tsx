import { List, Stack, Heading } from '@chakra-ui/react'
import Request from '@components/atoms/Cards/Request'
import React from 'react'

export default function RequestsList({
  requests,
  label
}: {
  requests: {
    href: string
    title: string
    description: string
  }[]
  label: string
}) {
  return (
    <Stack className="h-full" data-testid="requests-list">
      {requests.length > 0 && label && (
        <Heading as="h2" size="sm" className="mb-4">
          {label}
        </Heading>
      )}
      <div className="overflow-auto">
        {requests.length > 0 && (
          <List>
            {requests?.map((request) => (
              <div
                className="border-b border-black py-4 last:border-transparent"
                key={request.href}
              >
                <Request
                  href={request.href}
                  title={request.title}
                  description={request.description}
                />
              </div>
            ))}
          </List>
        )}
      </div>
    </Stack>
  )
}
