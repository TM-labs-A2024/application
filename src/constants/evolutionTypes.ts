export const evolutionTypes = [
  {
    id: '1',
    name: 'Evolución'
  },
  {
    id: '2',
    name: 'Hospitalización'
  },
  {
    id: '3',
    name: 'Alta'
  },
  {
    id: '4',
    name: 'Primera consulta'
  }
]
