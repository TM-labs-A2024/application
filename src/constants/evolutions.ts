export const evolution = {
  specialty: {
    description: '',
    id: '',
    name: ''
  },
  author: '',
  title: '',
  description: '',
  'content-format': '',
  created_at: '',
  payload: {
    type: {
      value: '',
      label: ''
    },
    reason: '',
    description: '',
    history: '',
    examination: '',
    summary: '',
    diagnostic: '',
    comments: ''
  }
}
