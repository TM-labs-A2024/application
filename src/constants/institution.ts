export const institution = {
  id: '3fa85f64-5717-4562-b3fc-2c963f66afa1',
  govId: '123456',
  name: 'Clínica Tex Mex',
  credentials: '14578245',
  address: 'Av. Fernandez Peña, 147.',
  email: 'test@test.com',
  phoneNumber: '+582742214441',
  institutionUser: {
    id: '',
    firstname: '',
    lastname: '',
    govId: '',
    birthdate: '',
    email: 'test@test.com',
    phoneNumber: '+582742214441',
    role: '',
    institutionId: ''
  }
}
