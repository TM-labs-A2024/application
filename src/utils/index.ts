import { formatDate } from './dates'
import { sendEmail, generateVerificationCode, setupEmailSending } from './email'
import { setupErrorNotification } from './notifications'
import { isWeb, isIOS, isAndroid, isMobile } from './platform'

export {
  isWeb,
  isIOS,
  isAndroid,
  isMobile,
  sendEmail,
  generateVerificationCode,
  setupEmailSending,
  setupErrorNotification,
  formatDate
}
