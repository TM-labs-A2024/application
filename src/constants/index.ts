import { doctors } from './doctors'
import { evolutionTypes } from './evolutionTypes'
import { institution } from './institution'
import { institutions } from './institutions'
import {
  ACCESS_REMOVED,
  FILTERS_APPLIED,
  ACCESS_REQUESTED,
  ATTACHMENT_DELETED,
  IMAGE_DELETED,
  ACCESS_DENIED,
  ACCESS_GRANTED,
  ACCESS_REMOVAL,
  GENERIC_NOTIFICATION,
  GENERIC_ERROR
} from './notifications'
import { nurse } from './nurses'
import { patient, sexOptions, statusOptions } from './patient'
import { patients } from './patients'
import { specialties, specialities } from './specialties'
import { specialtyData } from './specialtyData'

export {
  specialties,
  specialities,
  doctors,
  nurse,
  patient,
  sexOptions,
  statusOptions,
  specialtyData,
  evolutionTypes,
  patients,
  institution,
  institutions,
  ACCESS_REMOVED,
  FILTERS_APPLIED,
  ACCESS_REQUESTED,
  ATTACHMENT_DELETED,
  IMAGE_DELETED,
  ACCESS_DENIED,
  ACCESS_GRANTED,
  ACCESS_REMOVAL,
  GENERIC_NOTIFICATION,
  GENERIC_ERROR
}
