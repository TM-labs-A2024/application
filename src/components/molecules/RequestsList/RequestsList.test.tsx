import { doctors, specialties as specialtiesList } from '@src/constants'
import { render, screen } from '@test/utils/index'

import '@testing-library/jest-dom'
import RequestsList from './RequestsList'

const formatedRequests = doctors.map(({ id, firstname, lastname, specialties }) => ({
  href: `/institucion/medico/${id}`,
  title: `${firstname} ${lastname}`,
  description: `C.I: ${id},${specialties.map((specialty) => ' ' + specialtiesList.find((el) => el.id === specialty)?.name)}.`
}))

const props = {
  requests: formatedRequests,
  label: 'label'
}

describe('Molecules > RequestsList test', () => {
  test('The component shows the label if the requests list has at least one entry', () => {
    render(<RequestsList {...props} />)

    expect(screen.getByText(props.label)).toBeInTheDocument()
  })

  test('The quantity of children is the expected', () => {
    render(<RequestsList {...props} />)

    expect(screen.getAllByTestId('request-card')).toHaveLength(48)
  })

  test('Matches the snapshot', () => {
    render(<RequestsList {...props} />)

    expect(screen.getByTestId('requests-list')).toMatchSnapshot()
  })
})
