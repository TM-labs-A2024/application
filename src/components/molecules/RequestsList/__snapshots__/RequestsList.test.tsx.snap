// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Molecules > RequestsList test Matches the snapshot 1`] = `
<div
  class="chakra-stack h-full css-8g8ihq"
  data-testid="requests-list"
>
  <h2
    class="chakra-heading mb-4 css-9f6g39"
  >
    label
  </h2>
  <div
    class="overflow-auto"
  >
    <ul
      class="css-155za0w"
      role="list"
    >
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/1000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 1000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/2000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 2000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/3000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 3000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/4000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 4000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/5000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 5000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/6000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 6000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/7000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 7000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/8000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 8000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/9000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 9000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/10000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 10000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/11000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 11000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/12000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 12000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/13000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 13000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/14000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 14000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/15000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 15000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/16000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 16000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/17000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 17000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/18000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 18000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/19000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 19000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/20000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 20000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/21000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 21000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/22000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 22000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/23000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 23000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/24000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 24000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/25000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 25000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/26000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 26000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/27000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 27000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/28000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 28000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/29000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 29000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/30000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 30000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/31000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 31000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/32000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 32000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/33000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 33000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/34000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 34000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/35000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 35000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/36000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 36000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/37000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 37000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/38000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 38000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/39000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 39000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/40000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 40000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/41000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 41000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/42000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 42000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/43000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 43000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/44000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 44000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/45000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 45000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/46000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 46000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/47000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 47000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
      <div
        class="border-b border-black py-4 last:border-transparent"
      >
        <a
          class="chakra-link w-full css-spn4bz"
          data-testid="request-card"
          href="/institucion/medico/48000000"
        >
          <div
            class="flex flex-row items-center justify-between"
          >
            <div
              class="flex flex-col items-start gap-1"
            >
              <b
                class="chakra-text css-qqfgvy"
              >
                José Pérez
              </b>
              <p
                class="chakra-text css-0"
              >
                C.I: 48000000, undefined.
              </p>
            </div>
            <div />
          </div>
        </a>
      </div>
    </ul>
  </div>
</div>
`
